pyproject.toml

```diff
@@ -63,13 +69,29 @@ exclude = [
 line-length = 79
 indent-width = 4
 
+
 # Assume Python 3.9
 target-version = "py39"
 
 [tool.ruff.lint]
+pydocstyle.convention = "google"
 # Enable Pyflakes (`F`) and a subset of the pycodestyle (`E`)  codes by default.
-select = ["E", "F", "W"]
-ignore = []
+select = ["ALL"]
```

Comment:
Change for different PR.

Reply:
I'll remove this from the PR just before the merge. I want it to be linted by these rules.

Reply:
I'll remove this from the PR just before the merge. I want it to be linted by these rules.